src/macaron/config/defaults.ini

```diff
@@ -145,7 +145,12 @@ wrapper_files =
     mvnw
 
 [builder.maven.ci.build]
-github_actions = actions/setup-java
+github_actions =
+    actions/setup-java
+    # Parent project used in Maven-based projects of the Apache Logging Services.
+    apache/logging-parent/.github/workflows/build-reusable.yaml
+    # This action can be used to deploy artifacts to a JFrog artifactory server.
```

Comment:
Should this entry belong to `builder.maven.ci.deploy` instead?

Reply:
No, build and deploy GitHub Actions are obtained differently and from `builder.maven.ci` in `defaults.ini`.